### PR TITLE
Lint using pre-commit.

### PR DESCRIPTION
**Pull request by icemac**
- Remove `--no-flake8` option
- Remove unused [flake8] additional-plugins config option

Used in https://github.com/zopefoundation/Zope/pull/1224

### CODE REVIEW

**Comment by icemac** [comment]
@dataflake This PR is now ready to be reviewed. Additionally I'll try it out on a couple more repos.

**Comment by dataflake** [comment]
@icemac I added https://github.com/zopefoundation/meta/pull/265/commits/023baad475d047217f5f84b2ae68993cf88c9a66 to fix a typo that caused the manylinux failure at https://github.com/zopefoundation/zope.security/actions/runs/10474641317/job/29009385892

File: config/zope-product/tox.ini.j2 [hunk 1]
```diff
@@ -38,38 +38,7 @@ commands =
 %(line)s
 {% endfor %}
 
-{% include 'tox-release-check.j2' %}
```
Comment (dataflake) [inline config/zope-product/tox.ini.j2:41]: This line has to stay, right?

**Review by dataflake — CHANGES_REQUESTED** [review]
Looks like the zope-product tox template suffered from too many deletions

**Review by dataflake — APPROVED** [review]

File: config/zope-product/tox.ini.j2 (hunk 1 above)
Comment (icemac) [reply config/zope-product/tox.ini.j2:41]: No, the next line includes `tox-lint.j2` which already includes this file at the top, see config/default/tox-lint.j2

So either we have to remove it here or in `tox-lint.j2` but then add it in the places where `tox-lint.j2` is included.

**Pull request merged**